env0/data_configuration_variable.go

```diff
@@ -127,12 +127,12 @@ func dataConfigurationVariableRead(ctx context.Context, d *schema.ResourceData,
 	d.Set("is_sensitive", variable.IsSensitive)
 	d.Set("scope", variable.Scope)
 	d.Set("enum", variable.Schema.Enum)
-	if variable.Type == client.ConfigurationVariableTypeEnvironment {
+	if *variable.Type == client.ConfigurationVariableTypeEnvironment {
```

Comment:
what this change means?

Reply:
this (and all the other chnages to the variable client\resource test files) was so that I can omit none mandatory fields from the request
omitEmpty omits nullish values not false values, so inorder for it to work I changed the omittable values to pointers (so its null for nothing and if not it will be included)